### PR TITLE
Sf staging feedback - round 2

### PR DESCRIPTION
**Pull request by jamesmpigott**
#
### Description:
- Updates logo
- Restyle Org login in footer (make bigger)
- Add directions/how to get here field to all event forms, add to show page
- Update Accessibility information to be checkboxes
- Update event blueprints accordingly
- Add placeholder image functionality (given a basic place-hold.it image for now)
- Tweaks to image rendering to supply appropriately sized images and webp versions where possible
- Add directions field to org saved addresses
- Add max 50 characters for event title
- Minor accessibility updates

### Loom
https://www.loom.com/share/88a9871a2ed940b2a11bdce1d80ebeff

### Link to asana / trello card (if applicable)
https://app.asana.com/0/1207139965613063/1208330105745193/f

### Update 20/09/24
I've added [Event Reminders](https://app.asana.com/0/0/1208359291274530/f) - so when an event is 5 days away, an email (see task for content) is sent to the event author.

The event author is determined by the following:
- If the event has been uploaded via the dashboard/portal, it has a user associated with it, the email is sent to the user
- If the event has been uploaded via the public facing "Submit an event" form, provided the event has been approved+published, the submitter's details are stored in the event.

### CODE REVIEW

File: resources/views/events/preview.antlers.html [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
   <a class="c-event-card__link" href="{{ url }}">
     <figure class="c-event-card__image">
-      {{ if featured_image }}
-        {{ glide:featured_image width="834" height="556" }}<img src="{{ url }}" alt="{{ alt }}" />{{ /glide:featured_image }}
+    {{ img = featured_image ?? global:placeholder_image }}
+
+    {{ if img }}
+        <picture>
+            <source
+                src="{{ glide:img h="265" w="400" q="80" fit="contain" format="webp" }}"
```
Comment (joshtaylordev) [inline resources/views/events/preview.antlers.html:9]:
```suggestion
                srcset="{{ glide:img h="265" w="400" q="80" fit="contain" format="webp" }}"
```

File: resources/views/events/preview.antlers.html [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+                type="image/webp"
+            />
+            <source
+                src="{{ glide:img h="265" w="400" q="80" fit="contain" format="jpg" }}"
```
Comment (joshtaylordev) [inline resources/views/events/preview.antlers.html:13]:
```suggestion
                srcset="{{ glide:img h="265" w="400" q="80" fit="contain" format="jpg" }}"
```

File: resources/views/events/show.antlers.html [hunk 3]
```diff
@@ -91,12 +101,39 @@
   </aside>
 
   <div class="o-flow o-flow--l">
-    {{ if featured_image }}
-      {{ glide:featured_image width="1251" height="834" }}<img src="{{ url }}" alt="{{ alt }}" />{{ /glide:featured_image }}
+    {{ img = featured_image ?? global:placeholder_image }}
+    {{ if img }}
+        <picture>
+            <source
+                src="{{ glide:img h="550" w="821" q="80" fit="contain" format="webp" }}"
```
Comment (joshtaylordev) [inline resources/views/events/show.antlers.html:108]:
```suggestion
                srcset="{{ glide:img h="550" w="821" q="80" fit="contain" format="webp" }}"
```

File: resources/views/events/show.antlers.html [hunk 4]
```diff
@@ -91,12 +101,39 @@
...
+                type="image/webp"
+            />
+            <source
+                src="{{ glide:img h="550" w="821" q="80" fit="contain" format="jpg" }}"
```
Comment (joshtaylordev) [inline resources/views/events/show.antlers.html:112]:
```suggestion
                srcset="{{ glide:img h="550" w="821" q="80" fit="contain" format="jpg" }}"
```

File: resources/views/global/page-header/home.antlers.html [hunk 5]
```diff
@@ -14,11 +14,26 @@ <h1>{{ title }}</h1>
...
           {{ if featured_image }}
-            {{ glide:featured_image width="1251" height="834" }}
-              <figure class="c-page-header_content-image">
-                <img src="{{ url }}" alt="{{ alt }}" />
-              </figure>
-            {{ /glide:featured_image }}
+            <figure class="c-page-header_content-image">
+                <picture>
+                    <source
+                        src="{{ glide:featured_image h="490" w="730" q="80" fit="contain" format="webp" }}"
```
Comment (joshtaylordev) [inline resources/views/global/page-header/home.antlers.html:20]:
```suggestion
                        srcset="{{ glide:featured_image h="490" w="730" q="80" fit="contain" format="webp" }}"
```

File: resources/views/global/page-header/home.antlers.html [hunk 6]
```diff
@@ -14,11 +14,26 @@ <h1>{{ title }}</h1>
...
+                        type="image/webp"
+                    />
+                    <source
+                        src="{{ glide:featured_image h="490" w="730" q="80" fit="contain" format="jpg" }}"
```
Comment (joshtaylordev) [inline resources/views/global/page-header/home.antlers.html:24]:
```suggestion
                        srcset="{{ glide:featured_image h="490" w="730" q="80" fit="contain" format="jpg" }}"
```

**Review by joshtaylordev — CHANGES_REQUESTED** [review]

**Comment by jamesmpigott** [comment]
@joshtaylordev see updated loom.

File: resources/views/users/dashboard/loading-state.antlers.html
Comment (joshtaylordev) [inline resources/views/users/dashboard/loading-state.antlers.html:1]: https://www.loom.com/share/4ba0d1c612994be29cb47d119bc2f343?sid=d12b3167-575b-4260-8d56-36126bba3eac

File: resources/views/users/dashboard/my-events-create.antlers.html [hunk 7]
```diff
@@ -207,13 +176,13 @@
                 {{ partial:event-submission-form/text
```
Comment (joshtaylordev) [inline resources/views/users/dashboard/my-events-create.antlers.html:176]: I can't comment on the right section. But I noticed if there are errors with the submission the image gets lost

File: resources/views/users/dashboard/my-events.antlers.html
Comment (joshtaylordev) [inline resources/views/users/dashboard/my-events.antlers.html:1]: If the event's location is too long the buttons get squished

Screenshot 2024-09-24 at 10 39 49

**Review by joshtaylordev — CHANGES_REQUESTED** [review]

File: resources/views/users/dashboard/my-events-addresses.antlers.html [hunk 8]
```diff
@@ -0,0 +1,120 @@
+{{ partial:event-submission-form/section section_title="Location" }}
+    {{ slot:attributes }}
+        x-data="eventLocationForm"
+    {{ /slot:attributes }}
+
+
+    <div class="tw-flex tw-flex-col tw-gap-4 tw-border tw-border-brand-grey/30 tw-px-4 tw-py-4">
+        <h3 class="tw-text-2xl tw-mb-4">Saved Addresses</h3>
```
[the remark this answers is not in the document]
Comment (jamesmpigott) [reply resources/views/users/dashboard/my-events-addresses.antlers.html:8]: The user needs to be linked to an organisation - if you pull this branch the dev@ user is assigned to a test org I setup (titled Steadfast Collective)

File: resources/views/users/dashboard/my-events-addresses.antlers.html [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+                    </template>
+                </tbody>
+            </table>
+        </template>
+
+
+        <p class="tw-text-base tw-text-brand-grey/80">To add a saved address click on 'Addresses' on the menu at the top left of this page</p>
+    </div>
+
+    {{ partial:event-submission-form/text field_label="Search for a location..." field_name="location_autocomplete" }}
```
[the remark this answers is not in the document]
Comment (jamesmpigott) [reply resources/views/users/dashboard/my-events-addresses.antlers.html:74]: Dm'ed you :)

File: resources/views/users/dashboard/my-events-create.antlers.html (hunk 7 above)
Comment (jamesmpigott) [reply resources/views/users/dashboard/my-events-create.antlers.html:178]: Losing the image is normal behaviour - you can't pass an `old()` value into a file input (I think it's a browser thing) - there's a bunch of security concerns with it)

File: resources/views/users/dashboard/loading-state.antlers.html
Comment (joshtaylordev) [reply resources/views/users/dashboard/loading-state.antlers.html:1]: I think I was doing `npm run dev` which resulted in the above ^ 

But `npm run watch` fixes it so 👍

**Review by joshtaylordev — APPROVED** [review]

**Pull request merged**